task4/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+	"train_go_ep/task4/figure"
+	"train_go_ep/task4/person"
+	"train_go_ep/task4/utils"
+	"github.com/kyokomi/emoji"
+)
+
+func main() {
+	ivanIvanovDate := utils.DateParse("2005-Aug-10")
+	ivanIvanovSecondDate := utils.DateParse("2003-Aug-05")
+	ivanBbbbDate := utils.DateParse("2005-Aug-10")
+	artiomIvanovDate := utils.DateParse("2005-Aug-10")
+	artiomPetrovDate := utils.DateParse("2010-Aug-10")
+	arseniiAaaaDate := utils.DateParse("2005-Aug-10")
+	dimaFedorovDate := utils.DateParse("2004-Jun-01")
+	sahsaFedorovDate := utils.DateParse("2004-Jun-01")
+
+	p := person.People{
+		{"Ivan", "Ivanov", ivanIvanovDate},
```

Comment:
Usually, it's not the best idea to define structs in related way.

Reply:
You mean "name: value " syntax initializing? 
```
p := person.People{
		{FirstName: "Ivan", LastName: "Ivanov", BirthDay: ivanIvanovDate},
...
```

Reply:
@Vadimkatr  Yep, you should include field names as well, it improves readability )